packages/mail/templates/cf_comment_notification_new.html

```diff
@@ -13,41 +13,51 @@
+                    <p style="color:#282828;font-size:17px;line-height:24px;*|SG_FONT_STYLE_SANS_SERIF_REGULAR|*"><strong style="*|SG_FONT_STYLE_SANS_SERIF_MEDIUM|*">*|COMMENTER_NAME|*</strong> schreibt:</p>
+                    <div style="background-color:#ebf6e5;padding:10px;color:#282828;font-size:17px;line-height:24px;*|SG_FONT_STYLE_SERIF_REGULAR|*">*|CONTENT|*</div>
+                    <p style="color:#282828;font-size:17px;line-height:24px;*|SG_FONT_STYLE_SANS_SERIF_REGULAR|*"><a href="*|URL|*">Beitrag im Kontext auf republik.ch anzeigen</a>. Dort können Sie antworten und weiter debattieren.<br> 
                     </p>
                   </td>
                 </tr>
                 <tr>
                   <td style="padding:0 20px 20px 20px;">
-                    <p style="color:#282828;font-size:14px;line-height:20px;font-family:'GT-America-Standard-Regular', 'Helvetica-Neue-Regular', 'Arial-Regular', 'Roboto-Regular', sans-serif;">
+                    <p style="color:#282828;font-size:14px;line-height:20px;*|SG_FONT_STYLE_SANS_SERIF_REGULAR|*">
                       Sie erhalten diese E-Mail, weil Sie an der Debatte «<a href="*|DISCUSSION_URL|*">*|DISCUSSION_TITLE|*</a>» teilgenommen haben und/oder Benachrichtigungen für diese abonniert haben.
                     </p>
-                    <p style="color:#282828;font-size:14px;line-height:20px;font-family:'GT-America-Standard-Regular', 'Helvetica-Neue-Regular', 'Arial-Regular', 'Roboto-Regular', sans-serif;">
+                    <p style="color:#282828;font-size:14px;line-height:20px;*|SG_FONT_STYLE_SANS_SERIF_REGULAR|*">
                       Wollen Sie keine Benachrichtigungen zu dieser Debatte mehr erhalten? <a href="*|DISCUSSION_MUTE_URL|*">Jetzt deaktivieren</a>.<br>
                       Falls Sie gar keine Benachrichtigungen per E-Mail mehr erhalten möchten, können Sie dies in <a href="https://republik.ch/konto#benachrichtigungen">Ihrem Konto festlegen</a>.
                     </p>
                   </td>
                 </tr>
                 <tr>
                   <td style="padding:20px;">
-                    <p style="color:#282828;font-size:17px;line-height:24px;font-family:'GT-America-Standard-Regular', 'Helvetica-Neue-Regular', 'Arial-Regular', 'Roboto-Regular', sans-serif;">
-                      <img height="79" src="https://gallery.mailchimp.com/650c1ec9003c7d8567eef4c5e/images/feb6bdde-83da-4211-bcf8-b09d3d3d012a.png" style="border: 0px  ; width: 180px !important; height: 79px !important; margin: 0px;" width="180" alt="feb6bdde-83da-4211-bcf8-b09d3d3d012a.png">
+                    <p style="color:#282828;font-size:17px;line-height:24px;*|SG_FONT_STYLE_SANS_SERIF_REGULAR|*">
+                      <img height="79" src="https://cdn.republik.space/s3/republik-assets/assets/images/logo_republik_newsletter.png" style="border:0px;width:180px !important; height:79px !important;margin:0px;" width="180" alt="">
```

Comment:
Maybe create an environment [variable for CDN host like `ASSETS_SERVER_BASE_URL`](https://github.com/orbiting/republik-frontend/blob/master/.env.example#L4).